### ISSUE TITLE
Unable to update CIgroup query

### ISSUE THREAD

**Issue by PcoWegenenVerkeer** [taers232c/GAMADV-XTD3#152]
Hi,

I try the update method described here:
https://github.com/taers232c/GAMADV-XTD3/wiki/Cloud-Identity-Groups#manage-groups

With this kind of command: 
`gam update cigroup "test@verzendlijst.test.be" json file settings.json dynamic "user.organizations.exists(org, org.department=='test')"`

I get this response:
```
Command: /home/weynebe/bin/gamadv-xtd3/gam update cigroups test@verzendlijst.test.be json file settings.json >>>dynamic<<< "user.organizations.exists(org, org.department=='test')"

ERROR: Invalid argument
Help: Syntax in file /home/weynebe/bin/gamadv-xtd3/GamCommands.txt
Help: Documentation is at https://github.com/taers232c/GAMADV-XTD3/wiki
```

I updated to latest GAMADV-XTD3 6.03.14

Please help me out, according to wiki documentation update of "dynamic" should be possible.
Many thanks in advance!

**Comment by taers232c** [comment]
It's a bug, will be fixed later this morning.

Ross

**Comment by taers232c** [comment]
Fixed 6.03.15

**Comment by PcoWegenenVerkeer** [comment]
Many thanks!
